utils/tim_delay.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * tim_delay.h
+ *
+ *  Created on: 19-Feb-2021
+ *      Author: Sonal Shrivastava
+ */
+
+#ifndef UTILS_TIM_DELAY_H_
+#define UTILS_TIM_DELAY_H_
+
+void tim_init(void);
+void delay(uint32_t ms);
```

Comment:
Just to clarify: Even if you rename this file to `delay.h`, I still think you can keep the `delay()` function named as you have named it. That is, no need to rename it to `delay_delay()`... 🙂

Reply:
That was just for testing purposes, I removed the function. It is not needed anymore